### PR TITLE
Update getting-started.asciidoc

### PR DESCRIPTION
**Pull request by magnusram05**
hits.sort contain values not keys

      {
        "_index" : "bank",
        "_type" : "_doc",
        "_id" : "1",
        "_score" : null,
        "_source" : {
          "account_number" : *1*,
          "balance" : 39225,
          "firstname" : "Amber",
          "lastname" : "Duke",
          "age" : 32,
          "gender" : "M",
          "address" : "880 Holmes Lane",
          "employer" : "Pyrami",
          "email" : "amberduke@pyrami.com",
          "city" : "Brogan",
          "state" : "IL"
        },
        "sort" : [
          *1*
        ]
      },
      {
        "_index" : "bank",
        "_type" : "_doc",
        "_id" : "2",
        "_score" : null,
        "_source" : {
          "account_number" : *2*,
          "balance" : 28838,
          "firstname" : "Roberta",
          "lastname" : "Bender",
          "age" : 22,
          "gender" : "F",
          "address" : "560 Kingsway Place",
          "employer" : "Chillium",
          "email" : "robertabender@chillium.com",
          "city" : "Bennett",
          "state" : "LA"
        },
        "sort" : [
          *2*
        ]
      },

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md)?
- If submitting code, have you built your formula locally prior to submission with `gradle check`?
- If submitting code, is your pull request against master? Unless there is a good reason otherwise, we prefer pull requests against master and will backport as needed.
- If submitting code, have you checked that your submission is for an [OS that we support](https://www.elastic.co/support/matrix#show_os)?
- If you are submitting this code for a class then read our [policy](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md#contributing-as-part-of-a-class) for that.

### CODE REVIEW

File: docs/reference/getting-started.asciidoc [hunk 1]
```diff
@@ -788,7 +788,7 @@ As for the response, we see the following parts:
 * `hits` – search results
 * `hits.total` – total number of documents matching our search criteria
 * `hits.hits` – actual array of search results (defaults to first 10 documents)
-* `hits.sort` - sort key for results (missing if sorting by score)
+* `hits.sort` - sort value for results (missing if sorting by score)
```
Comment (cbuescher) [inline docs/reference/getting-started.asciidoc]: if you really want to be fussy, it could probably be "value of the sort key for each result"

**Review by cbuescher — CHANGES_REQUESTED** [review]
Hi @magnusram05, thanks for taking an interest in the getting-started docs. I left a small comment which might even improve upon your suggested edit. If you agree just push the change and I can run the tests and merge.

**Comment by javanna** [comment]
hi @magnusram05 do you plan on updating your PR based on the comment above?

**Comment by magnusram05** [comment]
Hi,

I will merge it ASAP.

Regards,
Ram

File: docs/reference/getting-started.asciidoc (hunk 1 above)
Comment (magnusram05) [reply docs/reference/getting-started.asciidoc]: I have made the changes.

**Comment by cbuescher** [comment]
@elasticmachine test this please

**Review by cbuescher — APPROVED** [review]
Thanks for making the change, LGTM.
I will merge this to 6.6 and later branches.

**Comment by cbuescher** [comment]
@elasticmachine update branch

**Comment by cbuescher** [comment]
@elasticmachine test this please

**Comment by cbuescher** [comment]
@magnusram05 thanks for the change, I merged it to all relevant branches.

**Comment by magnusram05** [comment]
Thanks. Appreciate it.

This is my first open source contribution. :)

Regards,
Ram

**Pull request merged**